scripts/exglobal_archive.sh

```diff
@@ -16,12 +16,7 @@ fi
 # CURRENT CYCLE
 APREFIX="${CDUMP}.t${cyc}z."
 ASUFFIX=${ASUFFIX:-${SUFFIX}}
-
-if [ "${ASUFFIX}" = ".nc" ]; then
-   format="netcdf"
-else
-   format="nemsio"
-fi
+format="netcdf"
```

Comment:
And this.

Reply:
Variable format is used locally. Therefore it should be ok not to change it.